pkg/config/otlp.go

```diff
@@ -100,6 +82,7 @@ func setupOTLPEnvironmentVariables(config Config) {
 	config.BindEnv(OTLPSection + ".metrics.sums.cumulative_monotonic_mode")
 	config.BindEnv(OTLPSection + ".metrics.summaries.mode")
 
-	// Debug setting
-	config.BindEnv(OTLPSection + ".debug.loglevel")
+	// Debug settings
+	config.BindEnv(OTLPSection + ".debug.loglevel") // Deprecated
+	config.BindEnv(OTLPSection + ".debug.verbosity")
```

Comment:
Is it expected to refer to `OTLPDebug` above and use `debug` here ?
Is there any change that `debug` key will change for `logging` later on ? In this case it might be interesting to use `OTLPDebug` instead of `debug` imho. Otherwise it should be ok (this is way more readable like this)👌